homeassistant/components/history.py

```diff
@@ -23,34 +24,20 @@
 DEPENDENCIES = ['recorder', 'http']
 
 CONFIG_SCHEMA = vol.Schema({
-    DOMAIN: vol.Schema({
-        CONF_EXCLUDE: vol.Schema({
-            vol.Optional(CONF_ENTITIES, default=[]): cv.entity_ids,
-            vol.Optional(CONF_DOMAINS, default=[]):
-                vol.All(cv.ensure_list, [cv.string])
-        }),
-        CONF_INCLUDE: vol.Schema({
-            vol.Optional(CONF_ENTITIES, default=[]): cv.entity_ids,
-            vol.Optional(CONF_DOMAINS, default=[]):
-                vol.All(cv.ensure_list, [cv.string])
-        })
-    }),
+    DOMAIN: cv.FILTER_SCHEMA,
 }, extra=vol.ALLOW_EXTRA)
 
 SIGNIFICANT_DOMAINS = ('thermostat', 'climate')
 IGNORE_DOMAINS = ('zone', 'scene',)
 
 
-def last_5_states(entity_id):
-    """Return the last 5 states for entity_id."""
-    entity_id = entity_id.lower()
-
-    states = recorder.get_model('States')
-    return recorder.execute(
-        recorder.query('States').filter(
-            (states.entity_id == entity_id) &
-            (states.last_changed == states.last_updated)
-        ).order_by(states.state_id.desc()).limit(5))
+def last_recorder_run():
```

Comment:
Please use hybrid properties instead and add this to the RecorderRun model: http://stackoverflow.com/a/14616299/646416

We then no longer need a helper method. (and yes, we should probably remove more of these helper methods in favor of that)